app/components/FormFields/SelectField.js

```diff
@@ -1,86 +1,162 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Select from 'react-select';
+import styled from 'styled-components';
 import AsyncSelect from 'react-select/lib/Async';
-import { gray, black, white, APP_FONT } from 'style-constants';
 
-import Label from './Label';
-import WarningMessage from './WarningMessage';
+import {
+  blue,
```

Comment:
I probably missed that during reviews at some point. If it is not too much work than it is better to rename those constants and describe them but role rather than actual color.
For example, if decide to change shadow color from blue to gree then the constant has to be renamed. But if it name `shadow-color` instead of `blue` then all we need to change is value in the constants file.